src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -441,7 +441,7 @@ internal NamespaceOrTypeOrAliasSymbolWithAnnotations BindNamespaceOrTypeOrAliasS
                 case SyntaxKind.TupleType:
                     {
                         var tupleTypeSyntax = (TupleTypeSyntax)syntax;
-                        return TypeWithAnnotations.Create(AreNullableAnnotationsEnabled(tupleTypeSyntax.CloseParenToken), BindTupleType(tupleTypeSyntax, diagnostics));
+                        return TypeWithAnnotations.Create(AreNullableAnnotationsEnabled(tupleTypeSyntax.CloseParenToken), BindTupleType(tupleTypeSyntax, diagnostics, basesBeingResolved));
```

Comment:
>basesBeingResolved [](start = 166, length = 18)

Does VB already do this or is it doing something else?

Reply:
`TypeBinder.BindTypeSyntax` is already doing this.

---
In reply to: [423253202](https://github.com/dotnet/roslyn/pull/44098#discussion_r423253202) [](ancestors = 423253202)